Formula/ffmpeg@2.8.rb

```diff
@@ -20,6 +20,7 @@ class FfmpegAT28 < Formula
 
   depends_on "freetype"
   depends_on "frei0r"
+  depends_on "jack"
```

Comment:
Are you sure this is still needed after the `revision` is bumped?

Reply:
This one seems to be a bit strange to me as the libavdevice dylib within ffmpeg actually has a hard compile-time and run-time dependency on `jack`. Not sure why this has been working until now.

Reply:
Sure, no problem.

Reply:
@tschoonj Thanks ❤️

Reply:
@tschoonj Pulled, `jack` isn't necessary, can remove this.